### ISSUE TITLE
TypeError: requires_grad_() takes 1 positional argument but 2 were given

### ISSUE THREAD

**Issue by Adlemann** [fastai/fastai#3224]
TypeError: requires_grad_() takes 1 positional argument but 2 were given

I have updated the fastai cores to latest from github, somehow im getting this error when im trying to activate training with TensorboardCallbacks. Is this a kind of bug or what?

I ve described the issue here: 
https://forums.fast.ai/t/tensorboard-issue-fastai-v2/85766

**Comment by FabianHertwig** [comment]
Should be fixed with #3184

You can install the latest version of fastai with:

```sh
git clone https://github.com/fastai/fastai
pip install -e "fastai[dev]"
```

**Comment by muellerzr** [comment]
@jph00 this can be closed I believe
